s3-source-connector/src/test/java/io/aiven/kafka/connect/s3/source/input/ByteArrayTransformerTest.java

```diff
@@ -16,18 +16,18 @@
 
 package io.aiven.kafka.connect.s3.source.input;
 
-import static io.aiven.kafka.connect.s3.source.config.S3SourceConfig.EXPECTED_MAX_MESSAGE_BYTES;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
```

Comment:
It's a pre-existing test, but just noticed we are still using junit here, it's only in one place can you replace assertArrayEquals with
assertThat(result).containsExactlyInAnyOrder(record);
or
assertThat(result).containsExactly(record);